library/std/arithmetic.qs

```diff
@@ -488,4 +488,74 @@ namespace Microsoft.Quantum.Arithmetic {
         }
     }
 
+    /// # Summary
+    /// Sets a zero-initialized little-endian register zs to the sum of
+    /// little-endian registers xs and ys using the carry-lookahead algorithm.
+    ///
+    /// # Description
+    /// Computes zs := xs + ys + zs[0] modulo 2ⁿ, where xs, ys, and zs are
+    /// little-endian registers, Length(xs) = Length(ys) ≤ Length(zs) = n (!!!),
```

Comment:
Why the (!!!)?

Reply:
There's a discrepancy between this doc comment and the code. I want code to accept longer zs but haven't done it yet.

Reply:
Relaxed condition.